tensorflow/lite/micro/test_helpers.h

```diff
@@ -161,6 +161,9 @@ const Model* GetSimpleModelWithSubgraphsAndIf();
 // of "if" subgraph overlaps with the input tensor 2 of subgraph 1.
 const Model* GetModelWithIfAndSubgraphInputTensorOverlap();
 
+// Returns a flatbuffer model with two subgraphs.
```

Comment:
nitpicking: may want to comment number of tensor and scratch buffer request here?